tuberculosis/lib/i18n/tubuddy_messages_nl.arb

```diff
@@ -0,0 +1,128 @@
+{
+  "@@locale": "nl",
+  "@@last_modified": "2018-05-23T14:46:44.436771",
+  "calendarTitle": "Kalender",
+  "@calendarTitle": {
+    "description": "Title for calendar tab",
+    "type": "text",
+    "placeholders": {}
+  },
+  "faqTitle": "FAQ",
+  "@faqTitle": {
+    "description": "Title for faq tab",
```

Comment:
FAQ tab

Reply:
Komen die descriptions ergens in de app terug? Volgens mij niet, dus dan is het niet echt belangrijk

Reply:
Die komen nergens in terug. Zijn puur voor het vertalen